### PR TITLE
Adding patch for default processing parameters set

### PR DESCRIPTION
**Pull request by josenavas**
@antgonza this adds the patch for adding the default parameters set.

### CODE REVIEW

**Comment by antgonza** [comment]
Failures but seems that is something with Travis, restarted the build.

**Comment by antgonza** [comment]
I think the problem is that the patch is waiting for user input.

**Comment by josenavas** [comment]
@antgonza it should be fixed now

**Comment by antgonza** [comment]
No luck ... :weary: seems like you are not testing this locally.

**Comment by josenavas** [comment]
I did tested locally, but there is a specific test case that I did not tested and it is the one happening on Travis. At this point I run out of ideas on how to solve this. Any ideas?? Note that at this point you can ad the default parameters set using the command, so if there are no ideas on how to solve this, use the command to add the default parameter, so this does not block anything...

File: qiita_db/support_files/patches/13.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+-- This file should have been empty as we only need 16.py but it will fail if
```
Comment (adamrp) [inline qiita_db/support_files/patches/13.sql]: Should this say "13.py"?

**Comment by antgonza** [comment]
@josenavas Should we close this?

**Pull request closed without merging**